android/guava/src/com/google/common/cache/CacheBuilder.java

```diff
@@ -42,8 +42,23 @@
 import org.checkerframework.checker.nullness.compatqual.NullableDecl;
 
 /**
- * A builder of {@link LoadingCache} and {@link Cache} instances having any combination of the
- * following features:
+ * A builder of {@link LoadingCache} and {@link Cache} instances.
+ *
+ * <h2>Prefer <a href="https://github.com/ben-manes/caffeine/wiki">Caffeine</a> over {@code
+ * common.cache}</h2>
+ *
+ * <p>The successor to {@code common.cache} is <a
+ * href="https://github.com/ben-manes/caffeine/wiki">Caffeine</a>. Its API is designed to make it a
+ * nearly drop-in replacement (though it require Java 8 APIs). (Its equivalent to {@code
```

Comment:
@cpovirk It looks to me like there's a spelling mistake here? Please correct me if I'm wrong.

> (though it require Java 8 APIs)

to

> (though it requires Java 8 APIs)

What do you think?

Reply:
Thanks, will fix now.

Reply:
Great, thanks for the quick response!